### PR TITLE
Add a page showing new firmware on the LVFS

### PR DESCRIPTION
**Pull request by hughsie**
I'm often asked this, and from a personal point of view it's easier in a list
than sorting through the event log and checking which firmware is in testing or
stable.

### CODE REVIEW

**Comment by hughsie** [comment]
Screenshot from 2019-03-13 10-32-39

File: app/templates/firmware-new.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "default.html" %}
+{% block title %}New Firmware{% endblock %}
+
+{% block content %}
+<h1>New Firmware</h1>
+<p>
+  This list show the latest public firmware being made available on the LVFS.
```
Comment (superm1) [inline app/templates/firmware-new.html]: Maybe worth mentioning it's only showing last 200…

File: app/templates/firmware-new.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <tr class="row">
+    <td class="col-sm-2">{{format_humanize_naturaltime(fwev.timestamp)}}</td>
+    <td class="col-sm-5">
+      {{fwev.fw.mds[0].developer_name_display}}
+      {{fwev.fw.mds[0].name_display}}
+    </td>
+    <td class="col-sm-3">{{fwev.fw.version_display}}</td>
+    <td class="col-sm-2">
+      <a href="/lvfs/device/{{fwev.fw.mds[0].guids[0].value}}"
+         class="btn btn-info btn-block">Details</a>
```
Comment (superm1) [inline app/templates/firmware-new.html]: A little asterisk or something to indicate which are in testing vs stable is probably good too.

File: app/views_firmware.py [hunk 3]
```diff
@@ -61,6 +61,25 @@ def firmware(show_all=False):
 def firmware_all():
     return firmware(True)
 
+@app.route('/lvfs/firmware/new')
+@app.route('/lvfs/firmware/new/<int:limit>')
+def firmware_new(limit=200):
+
+    # get a sorted list of vendors
+    fwevs = db.session.query(FirmwareEvent).\
+                order_by(FirmwareEvent.timestamp.desc()).\
```
Comment (superm1) [inline app/views_firmware.py]: So this means it's going to jump the list again when it goes from testing -> stable right? Expected that some get to jump twice?

File: app/templates/firmware-new.html (hunk 1 above)
Comment (hughsie) [reply app/templates/firmware-new.html]: `This list show the last {{limit}} public firmwares available on the LVFS.`?

File: app/views_firmware.py (hunk 3 above)
Comment (hughsie) [reply app/views_firmware.py]: I did ponder only including the ones just in stable -- most users won't know how to enable the testing remote.

Comment (superm1) [reply app/views_firmware.py]: I think that makes more sense too

File: app/templates/firmware-new.html (hunk 1 above)
Comment (superm1) [reply app/templates/firmware-new.html]: Yeah that's good

**Comment by hughsie** [comment]
Agreed, only public firmware works better.

**Pull request merged**